packages/view/backend/sirius-components-view-builder/src/main/java/org/eclipse/sirius/components/view/builder/generated/diagram/SelectionDialogTreeDescriptionBuilder.java

```diff
@@ -0,0 +1,74 @@
+/*******************************************************************************
+ * Copyright (c) 2023, 2024 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.view.builder.generated.diagram;
+
+/**
+ * Builder for SelectionDialogTreeDescriptionBuilder.
+ *
+ * @author BuilderGenerator
+ * @generated
+ */
+public class SelectionDialogTreeDescriptionBuilder {
```

Comment:
Same comment

Reply:
I was mistaken, it has already been rebased on top of the commit I was thinking about. There's nothing to change here